vclib-openid/src/commonTest/kotlin/at/asitplus/wallet/lib/oidc/OidcSiopIsoProtocolTest.kt

```diff
@@ -100,14 +96,11 @@ class OidcSiopIsoProtocolTest : FreeSpec({
             relyingPartyUrl = relyingPartyUrl,
         )
         val document = runProcess(
-            verifierSiop,
-            walletUrl,
-            OidcSiopVerifier.RequestOptions(
+            verifierSiop, walletUrl, OidcSiopVerifier.RequestOptions(
```

Comment:
Please keep the line breaks

Reply:
fixed in commit 6368aeb70b5da95469e5cd904a523f7e68d4bb34